Makefile

```diff
@@ -1,7 +1,7 @@
 darwin:
 	/bin/bash -c "mkdir -p bin/darwin"
-	/bin/bash -c "crystal build src/scry.cr --release -s --no-debug -o bin/darwin/scry"
+	/bin/bash -c "crystal build src/scry.cr -s --no-debug -o bin/darwin/scry"
 
 linux:
 	/bin/bash -c "mkdir -p bin/linux"
-	/bin/bash -c "crystal build src/scry.cr --release -s --no-debug -o bin/linux/scry"
```

Comment:
Why remove `--release` from the Makefile? Won't this make final compiled binaries larger and unoptimized?

Reply:
:ditto: